cpp/src/fil/fil.cu

```diff
@@ -344,15 +344,16 @@ void init_dense(const cumlHandle& h, forest_t* pf,
   *pf = f;
 }
 
-void from_treelite(const cumlHandle& handle, forest_t* pforest,
-                   ModelHandle model, const treelite_params_t* tl_params) {
+forest_t from_treelite(const cumlHandle& handle, forest_t* pforest,
```

Comment:
It is absolutely required that `from_treelite` returns `forest_t`? If yes, then `pforest` is unnecessary and should be dropped. However, it is better not to change the API.

Reply:
yes, I tried to pass the data without making this change and cython would not read and pass the right data in C++.